stdlib/Makefile

```diff
@@ -197,6 +201,15 @@ stdlib.p.cmxa: $(OBJS:.cmo=.p.cmx)
 sys.ml: sys.mlp ../VERSION
 	sed -e "s|%%VERSION%%|`sed -e 1q ../VERSION | tr -d '\r'`|" sys.mlp > $@
 
+# The user write stdlib.ml, but we don't use it as it to build
+# stdlib.cmo. First we do the following rewriting via an awk
+# preprocessor:
```

Comment:
I think as a result of changes, this comment is now in an odd place. Perhaps better if itappears before the definition of `OTHERS` above?

This first bit also isn't particularly clear: perhaps `stdlib.ml is pre-processed so that:`